charts/dotcms/values.yaml

```diff
@@ -1,15 +1,144 @@
+imagePullPolicy: IfNotPresent
+terminationGracePeriodSeconds: 10
+scaleDownBeforeUpgrade: false
+configVersion: 1
+replicas: 1
+resources:
+  requests:
+    cpu: 500m
+    memory: 5Gi
+  limits:
+    cpu: 2
+    memory: 5Gi
+javaMemory: 2
+
+# Ingress configuration (minimal defaults)
+ingress:
+  tlsSecretName: developer-certificate-secret
+  host: ""
+  hostSuffix: dotcms.local
+  # type: "nginx" or "alb"
+  type: "nginx"
+  alb:
+    securityGroups:
+      useDefaults: true
+      default: []
+      additional: []
+    hosts:
+      default:
+        enabled: true
+        hostSuffix: ".dotcms.cloud"
+        certificateArn: ""
+        sslPolicy: ""
+      additionalHosts: []
+      additionalCertificateArns: []
+      stickySessions:
+        enabled: false
+        duration: 18000
+      idleTimeout: 3600
+      accessLogs:
+        enabled: false
+        bucketOverride: ""
+        prefixOverride: ""
+      wafArn: ""
+
+# Database configuration (mandatory)
+database:
+  local:
+    enabled: true
+    image: pgvector/pgvector:pg16
+    resources:
+      requests:
+        cpu: 100m
+        memory: 3Gi
+      limits:
+        cpu: 500m
+        memory: 3Gi
+  # Leave empty if no override is needed
+  secretNameOverride: ""
+  host: "db"
+  port: 5432
+  name: dotcms
+
+# Opensearch configuration (mandatory)
+opensearch:
+  host:
+  endpointUrl: "{{ .Values.opensearch.protocol }}://{{ .Values.opensearch.host }}:{{ .Values.opensearch.port }}"
+  local:
+    enabled: true
+    image: opensearchproject/opensearch:1
+  port: 9200
+  protocol: "http"
+  javaOpts: "-Xmx1G"
+  authType: "BASIC"
+  secretNameOverride: ""
+  clusterIdOverride: ""
+
+# SMTP (Mail) configuration
+mail:
+  enabled: false
+  smtp:
+    host: ""
+    port: 587
+    auth: "true"
+    sslProtocols: "TLSv1.2"
+    secretNameOverride: ""
+
+# Redis configuration
+redis:
+  local:
+    enabled: true
+    image: redis:latest
+  host:
+  port: 6379
+  secretNameOverride: ""
+  password: ""
+  sslEnabled: true
+  sessionHost: "redis"
+  sessionPersistentPolicies: "DEFAULT"
+
+# Environment Variables Defaults (mandatory secrets)
+envVarsDefaults:
+  DB_USERNAME: "SECRET:{{ include \"dotcms.secret.env.name\" (dict \"Values\" .Values \"secretName\" \"database\" \"overridePath\" \"database.secretNameOverride\") }}:username"
+  DB_PASSWORD: "SECRET:{{ include \"dotcms.secret.env.name\" (dict \"Values\" .Values \"secretName\" \"database\") }}:password"
+  DB_DNSNAME: "{{ .Values.database.host | default \"db\" }}"
+  DB_BASE_URL: "{{ printf \"jdbc:postgresql://%s:%v/%s\" (.Values.database.host | default \"db\") (int .Values.database.port) (include \"dotcms.db.name\" .) }}"
+  DOT_ES_AUTH_BASIC_USER: "SECRET:{{ include \"dotcms.secret.shared.name\" (dict \"Values\" .Values \"secretName\" \"elasticsearch\") }}:username"
+  DOT_ES_AUTH_BASIC_PASSWORD: "SECRET:{{ include \"dotcms.secret.shared.name\" (dict \"Values\" .Values \"secretName\" \"elasticsearch\") }}:password"
```

Comment:
This construct is a little cleaner than having to escape the double qoutes

Minor thing, but don't like too much the need to escape the quotes here This could be made more cleanly with using printf and use of the "quote" function.  

DOT_ES_AUTH_BASIC_PASSWORD: {{ printf "SECRET:%s:password" (include "dotcms.secret.shared.name" (dict "Values" .Values "secretName" "elasticsearch")) | quote }}